main.js

```diff
@@ -144,6 +145,11 @@ module.exports = function(options) {
 		next();
 	});
 
+	app.use(function(req, res, next) {
+		res.locals.jsonLd = [nextJsonLd.webSite()];
+		next();
+	});
```

Comment:
good point quincy. If it's on by default then it adds extra processing to do for high volume apis such as preflight, which isn't very desirable.

I'm thinking of abstracting all the user-facing app stuff into something like n-html-app early next year, so at that point could decide to make it the default